librarian_server_scripts/librarian_change_transfer_status.py

```diff
@@ -0,0 +1,58 @@
+"""
+A command-line script, for use in containers, to change the behaviour
+of transfers to external librarians.
+"""
+
+import argparse as ap
+
+parser = ap.ArgumentParser(
+    description=(
+        "Change the status of an external librarian, to enable or disable transfers."
+    )
+)
+
+parser.add_argument(
+    "--librarian",
+    help="Name of the librarian to change the status of.",
+    type=str,
+    required=True,
+)
+
+parser.add_argument(
+    "--enable",
+    help="Enable the librarian.",
+    action="store_true",
+)
+
+parser.add_argument(
+    "--disable",
+    help="Disable the librarian.",
+    action="store_true",
+)
+
+
+def main():
+    args = parser.parse_args()
+
+    if args.enable and args.disable:
+        raise ValueError("Cannot enable and disable at the same time.")
+
+    if not args.enable and not args.disable:
+        raise ValueError("Must enable or disable.")
+
+    from librarian_server.database import get_session
+    from librarian_server.orm import Librarian
+
+    with get_session() as session:
+        librarian = (
+            session.query(Librarian).filter_by(name=args.librarian).one_or_none()
+        )
+        if librarian is None:
+            raise ValueError(f"Librarian {args.librarian} does not exist.")
+
+        if args.enable:
+            librarian.transfers_enabled = True
+        elif args.disable:
+            librarian.transfers_enabled = False
```

Comment:
What needs to happen if the user does not pass anything, or passes both enable and disable?

If it cannot happen then you can add a mutually exclusive group. And then it is a simple logical operation:

```python
librarian.transfers_enabled = args.enable and (not args.disable)
```

Reply:
It will accept a case where the input looks like:
```bash
librarian_change_transfer_status --librarian princeton --enable --disable
```
Eventually the status will change to `False`. However a mutually exclusive group will fail the command line when both options are given.

```python
group = parser.add_mutually_exclusive_group()
group.add_argument('-a', '--option_a', action='store_true', help='Option A')
group.add_argument('-b', '--option_b', action='store_true', help='Option B')
```
```bash
(slurmise) ip8725@RCO-VWVY2NNQLM:~/tmp$ python args.py -a -b
usage: args.py [-h] [-a | -b]
args.py: error: argument -b/--option_b: not allowed with argument -a/--option_a
```